pkgs/by-name/wg/wg-access-server/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, buildGoModule
+, buildNpmPackage
+, fetchFromGitHub
+, makeWrapper
+, iproute2
```

Comment:
Have not been able to test, is iproute2 enough? I think it requires both iptables and ip (iproute2).

Reply:
It only need iptables in its path and iproute2 isn't required.